packages/worksheet/public/locales/en/worksheet.json

```diff
@@ -45,5 +45,8 @@
   "ENTER_WORKSHEET_DETAILS": "Enter Worksheet Details",
   "SHOW_QUESTIONS_BASED_ON": "Show questions based on",
   "PLEASE_SELECT_LIMIT": "Please select limit",
-  "QUESTION_NOT_FOUND": "No question found for this filter"
+  "QUESTION_NOT_FOUND": "No question found for this filter",
+  "SEARCH_QUESTIONS": "Search Questions",
+  "NEW_QUESTIONS_ADDED": "New Questions Added",
```

Comment:
review sentence case

Reply:
This is we manage by CSS like capabilities